Library/Formula/racket.rb

```diff
@@ -28,17 +31,43 @@ def install
       system "make"
       system "make", "install"
     end
+
+    # configure racket's package tool (raco) to do the Right Thing
+    # see: http://docs.racket-lang.org/raco/config-file.html
+    inreplace etc/"racket/config.rktd" do |s|
+        s.gsub!(
+            /\(bin-dir\s+\.\s+"#{Regexp.quote(bin)}"\)/,
+            "(bin-dir . \"#{HOMEBREW_PREFIX}/bin\")",
+        )
+        s.gsub!(
+            /\n\)$/,
+            "\n      (default-scope . \"installation\")\n)",
+        )
+    end
```

Comment:
We don't want to have to carry fairly hacky patches like this long-term as they will break. We're a big enough package manager that we should work with upstream to find out a mutually appropriate solution.